### ISSUE TITLE
Voice Over - Button remove from org unit is reading as time

### ISSUE THREAD

**Issue by vanpham-sw** [shopware/platform#7816]
### PHP Version

8.2.25

### Shopware Version

6.7

### Affected area / extension

Extension:Commercial

### Actual behaviour

When using Voice Over to read remove address or employee buttons, they are reading with x (time) button, which is confusing.  

https://github.com/user-attachments/assets/84e0f65c-3531-4de8-ac40-de5b7f4788f7

### Expected behaviour

Remove button should be read as "remove button"

### How to reproduce

1. Edit an org unit that has employee and address assigned
2. Use Voice Over to move to remove button in employee or address section
